libbeat/docs/loggingconfig.asciidoc

```diff
@@ -27,6 +27,7 @@ logging.files:
   path: /var/log/mybeat
   name: mybeat.log
   keepfiles: 7
+  permissions: 0644
```

Comment:
The users will be required to start the value with a leading 0 in order for it to be interpreted as octal in YAML.